### ISSUE TITLE
generator-electrode install fails with npm@^5

### ISSUE THREAD

**Issue by swashcap** [electrode-io/electrode#577]
## Type of issue

🐛 _Bug!_

## My environment

* *OS version/details*: macOS 10.12.6
* *Node version:* 8.5.0
* *npm version:* 5.4.2
* *Version of yo :* 2.0.0

## Expected behavior

Running `npm install` should install dependencies in a Yeoman generated electrode application

## Current behavior

Running `npm install` yields this error (see output below).

## Steps to reproduce the behavior

```shell
$ nvm install 8.5.0
$ npm i -g npm@5.4.2 yo generator-electrode
$ yo electrode
# go through steps, accept defaults...
```

## Command line output

Relevant section:

```shell
> electrode-archetype-opt-react@0.1.1 preinstall /Users/c0r00jl/Projects/store-map/node_modules/electrode-archetype-opt-react
> node optional-check.js

npm ERR! Cannot read property 'pause' of undefined

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/swashcap/.npm/_logs/2017-09-21T21_01_57_511Z-debug.log
```

Full error log: https://gist.github.com/swashcap/dcad6b015a891f71de7534690c692092

**Comment by gkarwchan** [comment]
this is not an electrode bug.
It is related to npm latest version.
There is a bug in [npm that is still open](https://github.com/npm/npm/issues/18520).
To fix it, downgrade to npm 5.3.0

**Comment by swashcap** [comment]
@gkarwchan Ah, gotcha. Closing this, I’ll keep an eye on that issue.
